src/utils/helpers/list/buildQueryFilters.ts

```diff
@@ -20,7 +23,65 @@ export default function buildQueryFilters(filters: BasicStringObject) {
   return graphQLfilter
 }
 
+const genericTypes: { computeFilter: GenericTypesMethod; columns: string[] }[] = [
+  // NUMBER TYPE
+  {
+    computeFilter: (filterString: string) => {
+      const [fromNumber, toNumber] = filterString.split(':')
+      const greaterThanOrEqualTo = fromNumber ? fromNumber : undefined
+      const lessThanOrEqualTo = toNumber ? toNumber : undefined
+
+      return { greaterThanOrEqualTo, lessThanOrEqualTo }
+    },
+    columns: [
+      'reviewAssignedNotStartedCount',
+      'reviewAssignedCount',
+      'reviewAvailableForSelfAssignmentCount',
+      'reviewDraftCount',
+      'reviewPendingCount',
+      'reviewChangeRequestCount',
+      'reviewSubmittedCount',
+      'assignReviewerAssignedCount',
+      'assignReviewersCount',
+      'assignCount',
```

Comment:
These will need to change when we make the change to return enums from application_list right?

Reply:
yeah, and it sounds like we dont' need a number type, although it could be useful somewhere ? (number of reviewers ?) or number of change request iterations ?